tests/main/lxd/task.yaml

```diff
@@ -34,8 +48,12 @@ restore: |
         lxd.lxc stop $cont_name --force
         lxd.lxc delete $cont_name
     done
+    snap remove --purge lxd
+    snap remove --purge lxd-demo-server
 
     "$TESTSTOOLS"/lxd-state undo-mount-changes
+    snap unset system experimental.refresh-app-awareness
+    systemctl --user stop dbus.service || true
```

Comment:
does this needs the same check about USER as the other test?

Reply:
No, because this test does everything as the root user. I've added a code comment to explain this now.